lib/std/os.zig

```diff
@@ -6056,30 +6056,39 @@ pub fn sendto(
+            .CONNREFUSED => return error.ConnectionRefused,
             .CONNRESET => return error.ConnectionResetByPeer,
             .DESTADDRREQ => unreachable, // The socket is not connection-mode, and no peer address is set.
             .FAULT => unreachable, // An invalid user space address was specified for an argument.
+            .HOSTUNREACH => return error.NetworkUnreachable,
             .INTR => continue,
-            .INVAL => return error.UnreachableAddress,
+            .INVAL => return error.UnreachableAddress, // (AF_UNIX) incorrect socket family given, or socket in invalid state
             .ISCONN => unreachable, // connection-mode socket was connected already but a recipient was specified
+            .LOOP => return error.SymLinkLoop,
             .MSGSIZE => return error.MessageTooBig,
+            .NAMETOOLONG => return error.NameTooLong,
+            .NETDOWN => return error.NetworkSubsystemFailed,
+            .NETUNREACH => return error.NetworkUnreachable,
+            .NFILE => return error.ResourceLimitReached, // The system-wide limit on the total number of open files has been reached.
             .NOBUFS => return error.SystemResources,
+            .NOENT => return error.FileNotFound,
             .NOMEM => return error.SystemResources,
+            .NOTCONN => return error.SocketNotConnected,
+            .NOTDIR => return error.NotDir,
             .NOTSOCK => unreachable, // The file descriptor sockfd does not refer to a socket.
             .OPNOTSUPP => unreachable, // Some bit in the flags argument is inappropriate for the socket type.
-            .PIPE => return error.BrokenPipe,
-            .AFNOSUPPORT => return error.AddressFamilyNotSupported,
-            .LOOP => return error.SymLinkLoop,
-            .NAMETOOLONG => return error.NameTooLong,
-            .NOENT => return error.FileNotFound,
-            .NOTDIR => return error.NotDir,
-            .HOSTUNREACH => return error.NetworkUnreachable,
-            .NETUNREACH => return error.NetworkUnreachable,
-            .NOTCONN => return error.SocketNotConnected,
-            .NETDOWN => return error.NetworkSubsystemFailed,
+            .PERM => return error.PermissionDenied, // The sender passed invalid credentials in the struct ucred.
+            .PIPE => return error.BrokenPipe, // Remote socket was closed on a stream socket.  If enabled, a SIGPIPE is sent as well.
+            .PROTONOSUPPORT => return error.ProtocolNotSupported, // (AF_UNIX) Passed protocol is not AF_UNIX.
+            .PROTOTYPE => return error.SocketTypeMismatch, // (AF_UNIX) Remote socket does not match the local socket type (SOCK_DGRAM versus SOCK_STREAM).
+            .SOCKTNOSUPPORT => return error.SocketTypeNotSupported, // (AF_UNIX) Unknown socket type.
+            .SRCH => return error.ProcessNotFound, // (AF_UNIX) While sending an ancillary message containing credentials (SCM_CREDENTIALS), the caller specified a PID that does not match any existing process.
```

Comment:
Can you make this text shorter?

Reply:
I don't trust myself. This is the first time I heard of those signals.